gotechnica/platform-2020
# Add Frontend Nav Bar

## Description
This PR encompasses completing our navigation bar header, such that navigating to the different platform pages are easily accessible via the nav bar.

## Steps to test
1. Compile the frontend web app
2. Go to the home page (should load automatically) and check the nav bar displays properly
3. Verify the nav bar is mobile responsive by resizing your browser window.

Trello ticket: https://trello.com/c/3ip3J2NM
